src/pages/card/view/index.tsx

```diff
@@ -3,11 +3,13 @@ import styled from 'styled-components';
 import { CardPreview, cardModel } from '@box/entities/card';
 import { ContentCenteredTemplate, UserCard } from '@box/ui';
 import { Helmet } from 'react-helmet-async';
+import { Link } from 'react-router-dom';
 import { useStart, withStart } from '@box/lib/page-routing';
 import { useStore } from 'effector-react/ssr';
 
 import * as model from './model';
-import { avatarUri } from '../../shared/constants';
+import { avatarUri } from '../../../shared/constants';
+import { paths } from '../../paths';
```

Comment:
```suggestion
import { avatarUri } from '@box/shared/constants';
import { paths } from '@box/pages/paths';
```

nitpick: тут кажется тоже лучше абсолютный путь

Reply:
Только за, просто я этот код не трогал, лишь переименовал)

Reply:
Хотя к `paths` оставил бы всеж относительный импорт, т.к. это +- внутренний ресурс